README: protobuf dependency example does not build

### What version of rules_go are you using?
v0.22.4

### What version of gazelle are you using?
v0.20.0

### What version of Bazel are you using?
3.0.0

### Does this issue reproduce with the latest releases of all the above?
Yes

### What operating system and processor architecture are you using?
Ubuntu 18.04, Linux studio 4.19.84-microsoft-standard #1 SMP Wed Nov 13 11:44:37 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

### Any other potentially useful information about your toolchain?
Running under Windows 10 - WSL 2

### What did you do?
Copied proto dependency example from README:
```bzl
load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")

http_archive(
    name = "com_google_protobuf",
    sha256 = "9748c0d90e54ea09e5e75fb7fac16edce15d2028d4356f32211cfa3c0e956564",
    urls = ["https://github.com/protocolbuffers/protobuf/archive/v3.11.4.zip"],
)

load("@com_google_protobuf//:protobuf_deps.bzl", "protobuf_deps")

protobuf_deps()
```

### What did you expect to see?
Build succeeds

### What did you see instead?
```
$ make gazelle
bazel run //:gazelle
ERROR: error loading package '': Every .bzl file must have a corresponding package, but '@com_google_protobuf//:protobuf_deps.bzl' does not have one. Please create a BUILD file in the same or any parent directory. Note that this BUILD file does not need to do anything except exist.
INFO: Elapsed time: 0.454s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)
Makefile:17: recipe for target 'gazelle' failed
make: *** [gazelle] Error 1
```

--- a/README.rst
+++ b/README.rst
@@ -426,6 +426,7 @@ To generate code from protocol buffers, you'll need to add a dependency on
     http_archive(
         name = "com_google_protobuf",
         sha256 = "9748c0d90e54ea09e5e75fb7fac16edce15d2028d4356f32211cfa3c0e956564",
+        strip_prefix = "protobuf-3.11.4",
         urls = ["https://github.com/protocolbuffers/protobuf/archive/v3.11.4.zip"],
     )
 

--- a/go/dependencies.rst
+++ b/go/dependencies.rst
@@ -130,6 +130,7 @@ the execution platform, too.
     http_archive(
         name = "com_google_protobuf",
         sha256 = "9748c0d90e54ea09e5e75fb7fac16edce15d2028d4356f32211cfa3c0e956564",
+        strip_prefix = "protobuf-3.11.4",
         urls = ["https://github.com/protocolbuffers/protobuf/archive/v3.11.4.zip"],
     )
 

